src/DynamoSandbox/Program.cs

```diff
@@ -106,12 +106,19 @@ private static void MakeStandaloneAndRun(string commandFilePath, out DynamoViewM
                     DynamoModel = model
                 });
 
-            DynamoModel.RequestMigrationStatusDialog -= MigrationStatusDialogRequested;
-
             var view = new DynamoView(viewModel);
+            view.Loaded += ViewLoaded;
 
             var app = new Application();
             app.Run(view);
+
+            DynamoModel.RequestMigrationStatusDialog -= MigrationStatusDialogRequested;
+        }
+
+        static void ViewLoaded(object sender, RoutedEventArgs e)
+        {
+            DynamoModel.OnRequestMigrationStatusDialog(new SettingsMigrationEventArgs(
+                        SettingsMigrationEventArgs.EventStatusType.End));
```

Comment:
Please don't do this, it's a weird behavior to go from `Program` into `DynamoModel` and then rely on it to call back into `Program` to do certain thing (it's like *can you remind me tomorrow to remind you to merge this PR?*). Instead of doing that, do this:

```csharp
private static void MakeStandaloneAndRun()
{
    view.Loaded += (sender, e) => CloseMigrationWindow();
}

void CloseMigrationWindow()
{
    if (migrationWindow != null) {
        migrationWindow.Close();
        migrationWindow = null;
    }
}
```